libmscore/style.h

```diff
@@ -1096,6 +1096,94 @@ enum class Sid {
       STYLES
       };
 
+// Internal resolution factor that multiplies values originally in points
+static constexpr qreal DPI_F = 5; // screen resolution factor
```

Comment:
I separated this out for readability.  In a future PR I will be adding one more constexpr here for MSC_F.